streams/src/main/java/org/apache/kafka/streams/kstream/Suppressed.java

```diff
@@ -136,12 +129,10 @@ static StrictBufferConfig unbounded() {
      * @param bufferConfig A configuration specifying how much space to use for buffering intermediate results.
      *                     This is required to be a "strict" config, since it would violate the "final results"
      *                     property to emit early and then issue an update later.
-     * @param <K> The key type for the KTable to apply this suppression to. "Final results" mode is only available
-     *           on Windowed KTables (this is enforced by the type parameter).
      * @return a "final results" mode suppression configuration
      */
-    static <K extends Windowed> Suppressed<K> untilWindowCloses(final StrictBufferConfig bufferConfig) {
-        return new FinalResultsSuppressionBuilder<>(bufferConfig);
+    static Suppressed<Windowed> untilWindowCloses(final StrictBufferConfig bufferConfig) {
```

Comment:
This is the other half of allowing to use this class without redundant generic type parameters.